chunker/README.md

```diff
@@ -44,16 +44,18 @@ When using`ChunkerAnnotator`, the models are loaded automatically from the direc
 Thus, to use your own models (maybe you need to train your models first; please read the following section `Training`), simply place them in this directory and they will be loaded; otherwise, the model version
 specified in this project's `pom.xml` file will be loaded from the Maven repository and used.
 
-**Note** : To use your own models, **exclude** the `illinois-chunker-model` artifact from the `cogcomp-chunker` dependency in your `pom.xml` to avoid potential conflicts.
+**Note** : To use your own models (either you're retraining it or using it), **exclude** the `illinois-chunker-model` artifact from the `cogcomp-chunker` dependency in your `pom.xml` to avoid potential conflicts.
 
 ## Training
-The class [`ChunkerTrain`](src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java) contains a main method that can be used to
-train the models for a Chunker provided you have access to the necessary training data. It can be called from the top-level
+1. If you just want to retrain the chunker models using your data, i.e., without modifying [chunk.lbj](src/main/lbj/chunk.lbj), then please read the following:
```

Comment:
Instead of "without modifying chunk.lbj", I would say "without modifying the features of the model", to better match what you have below.